### PR TITLE
Use prepared statements in Datastax C*

### PR DESCRIPTION
**Pull request by emilssolmanis**

### CODE REVIEW

**Comment by emilssolmanis** [comment]
**NOTE**: there's also a variant of `prepareAsync` which returns a `ListenableFuture`, as it stands the impl does the `prepare` calls sync. I don't think that's an issue since it's a one-off thing, but if you think it is, feel free to flag it.

File: atlas-cassandra/src/main/java/org/atlasapi/content/CassandraEquivalentContentStore.java [hunk 1]
```diff
@@ -273,46 +292,44 @@ private void deleteStaleRows(EquivalenceGraph updated, ImmutableSet<EquivalenceG
...
-        session.execute(batch(deletes.toArray(new RegularStatement[deletes.size()]))
-                .setConsistencyLevel(writeConsistency));
+
+        BatchStatement batchStatement = new BatchStatement();
+        batchStatement.setConsistencyLevel(writeConsistency);
+
+        batchStatement.addAll(created.stream()
+                .flatMap(graph -> graph.getEquivalenceSet().stream())
+                .map(elem -> rowDelete.bind()
+                        .setLong("setId", id)
```
Comment (tfm) [inline atlas-cassandra/src/main/java/org/atlasapi/content/CassandraEquivalentContentStore.java]: moved from constants to in-line strings?

**Comment by tfm** [comment]
only one minor thing, otherwise :+1:

**Comment by emilssolmanis** [comment]
@tfm not really, the strings are arg placeholder names, can make them all constants if you feel like that'll be better.

**Pull request merged**